### PR TITLE
Working Guessing Game

### PR DESCRIPTION
**Pull request by omerltf**
Version 1.0 of my guessing game

### CODE REVIEW

File: NumberGuess/Program.cs [hunk 1]
```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NumberGuess {
+    class Program {
+        static int maxPlayers = 2;
```
Comment (appacademy-jchurchill) [inline NumberGuess/Program.cs]: Typically class fields would start with an uppercase letter like this: `MaxPlayers`.

File: NumberGuess/Program.cs [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        /// Method to have user input their name
+        /// </summary>
+        /// <param name="playerNumber">Which player is inputting their name?</param>
+        /// <returns>Player's name as a string</returns>
+        public static string GetPlayerName(int playerNumber) {
+            while (true) {
+                Console.WriteLine();
+                Console.Write("Player {0}: Please enter your name. ", playerNumber);
+                string nameInput = Console.ReadLine();
+                if (String.IsNullOrEmpty(nameInput)) {
```
Comment (appacademy-jchurchill) [inline NumberGuess/Program.cs]: Prefer to use `string.IsNullOrEmpty()`.

File: NumberGuess/Program.cs [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+            }
+        }
+
+        /// <summary>
+        /// Method to have user input their number
+        /// </summary>
+        /// <param name="playerName">Which player is inputting their number?</param>
+        /// <param name="message">Message to display on console</param>
+        /// <returns>The integer input by the player</returns>
+
```
Comment (appacademy-jchurchill) [inline NumberGuess/Program.cs]: Kill this whitespace.

File: NumberGuess/Program.cs [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+        public static int GetNumber(string playerName, string message) {
+            const int minimumValue = 0;
+            const int maximumValue = 100;
+
+            //loop makes sure no bogey values are input
+            while (true) {
+                Console.WriteLine("{0}: {1}", playerName, message);
+                string temp_input = Console.ReadLine();
```
Comment (appacademy-jchurchill) [inline NumberGuess/Program.cs]: Rename this variable to `tempInput`.

File: NumberGuess/Program.cs [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+                int input = 0 ;
+                Int32.TryParse(temp_input,out input);
```
Comment (appacademy-jchurchill) [inline NumberGuess/Program.cs]: Prefer to use the `int` keyword here.

File: NumberGuess/Program.cs [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+                    answer = selectedOption.KeyChar - '0';
+                    if (!(answer > 1 && answer <= maxPlayers)) {
+                        Console.WriteLine("Incorrect input. Please try again.");
+                        continue;
+                    }
+                    else {
+                        break;
+                    }
+                }
+                string[] PlayerNames = new string[answer];
```
Comment (appacademy-jchurchill) [inline NumberGuess/Program.cs]: Rename to `playerNames`.

**Review by appacademy-jchurchill — CHANGES_REQUESTED** [review]
Looks great! Just some minor tweaks to do.

File: NumberGuess/Program.cs (hunk 1 above)
Comment (omerltf) [reply NumberGuess/Program.cs]: Fixed

File: NumberGuess/Program.cs (hunk 2 above)
Comment (omerltf) [reply NumberGuess/Program.cs]: Fixed

File: NumberGuess/Program.cs (hunk 3 above)
Comment (omerltf) [reply NumberGuess/Program.cs]: Fixed

File: NumberGuess/Program.cs (hunk 4 above)
Comment (omerltf) [reply NumberGuess/Program.cs]: Fixed

File: NumberGuess/Program.cs (hunk 6 above)
Comment (omerltf) [reply NumberGuess/Program.cs]: Fixed

File: NumberGuess/Program.cs (hunk 5 above)
Comment (omerltf) [reply NumberGuess/Program.cs]: Fixed

**Comment by appacademy-jchurchill** [comment]
Looks great! :shipit:

**Pull request merged**